terminal.go

```diff
@@ -631,6 +639,25 @@ func (g *LightningTerminal) start() error {
 	// We can now set the status of LiT as running.
 	g.statusMgr.SetRunning(subservers.LIT)
 
+	// Spawn a goroutine that will log any non-critical errors from
+	// sub systems, until we receive the main shutdown signal.
+	go func() {
+		for {
+			select {
+			case err := <-g.nonCriticalErrQueue.ChanOut():
+				log.Errorf("Received error from subsystem, "+
```

Comment:
If we're just logging the errors, shouldn't we instead just remove the error channel from the account service and turn all places where we would send to that channel into a log call?

Reply:
Made this a callback instead :)!